sdk/communication/Azure.Communication.CallingServer/src/Generated/CallingServerModelFactory.cs

```diff
@@ -12,7 +12,7 @@
 namespace Azure.Communication.CallingServer
 {
     /// <summary> Model factory for models. </summary>
-    public static partial class CallAutomationModelFactory
+    public static partial class CallingServerModelFactory
```

Comment:
This package is beta, I believe this name change is OK

Reply:
And its previous name collides with the one in `sdk\communication\Azure.Communication.CallAutomation`